linera-sdk/src/service/conversions_from_wit.rs

```diff
@@ -34,7 +34,20 @@ impl From<wit_system_api::HashedBlob> for HashedBlob {
 
 impl From<wit_system_api::BlobId> for BlobId {
     fn from(blob_id: wit_system_api::BlobId) -> Self {
-        BlobId(blob_id.inner0.into())
+        match blob_id.blob_type {
+            wit_system_api::BlobType::Data => Self {
+                hash: blob_id.hash.into(),
+                blob_type: blob_id.blob_type.into(),
```

Comment:
Why the `match`? Won't all `match` arms be identical?

Reply:
True 😅